### PR TITLE
metadata: enable mapping for copyrights and alt_text

### PR DESCRIPTION
**Pull request by vied12**
ping @akintolga @petrjasek :bowtie:

### CODE REVIEW

**Comment by petrjasek** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

File: superdesk/metadata/item.py [hunk 1]
```diff
@@ -108,17 +108,14 @@
     # Copyright Information
     'usageterms': {
         'type': 'string',
-        'mapping': not_analyzed,
         'nullable': True,
     },
     'copyrightnotice': {
         'type': 'string',
```
Comment (mdhaman) [inline superdesk/metadata/item.py]: What sort of data will be there in `copyrightnotice`? 
Are we going to search on `copyrightnotice`?

**Comment by mdhaman** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Comment by akintolga** [comment]
:+1: Looks good, merging...

**Pull request merged**